_posts/2020-07-23-hey-hey-hey-its-started.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+header_image: /img/posts/hey-hey-hey-its-started/header.jpg
+title: "Hey hey hey it's started"
+author: Mauro Servienti
+synopsis: "When using NServiceBus there are cases in which there is the need to perform actions when the endpoint is started. This can become cumbersome when using the .NET Core Generic Host infrastructure. NServiceBus.Extensions.EndpointStarted comes to the rescue, and allows to register a simple callback function during the endpoint configuration phase."
```

Comment:
```suggestion
synopsis: "When using NServiceBus, sometimes there is a need to perform actions when the endpoint is started. This can become cumbersome when using the .NET Core Generic Host infrastructure. NServiceBus.Extensions.EndpointStarted comes to the rescue, and allows to register a simple callback function during the endpoint configuration phase."
```